src/app/shared/context.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from '@angular/core';
+import { Context } from './../models/context';
+import { DummyService } from './../dummy/dummy.service';
+import { Router } from '@angular/router';
+import { Broadcaster } from '../shared/broadcaster.service';
+
+
+/*
+ * A shared service that manages the users current context. The users context is defined as the
+ * entity (user or org) and space that they are operating on.
+ * 
+ */
+@Injectable()
+export class ContextService {
+
+    public current: Context;
```

Comment:
Another style point. We use 2 space indents in most places, expect where a deeper indent improves readability.